source/server/http/admin.cc

```diff
@@ -541,9 +541,10 @@ Http::Code AdminImpl::handlerConfigDump(absl::string_view, Http::HeaderMap& resp
     auto& any_message = *(dump.add_configs());
     any_message.PackFrom(*message);
   }
+  const auto redacted = MessageUtil::redact(dump);
```

Comment:
@mergeconflict like here, no reason to copy because copy already happened above.

Reply:
You are absolutely right, I'll make the change :)